content/admin/github-actions/enabling-github-actions-for-github-enterprise-server/enabling-github-actions-with-minio-storage.md

```diff
@@ -12,22 +12,25 @@ topics:
   - Storage
 redirect_from:
   - /admin/github-actions/enabling-github-actions-with-minio-gateway-for-nas-storage
-shortTitle: MinIO Gateway for NAS storage
+  - /admin/github-actions/enabling-github-actions-for-github-enterprise-server/enabling-github-actions-with-minio-gateway-for-nas-storage
+shortTitle: MinIO storage
 ---
 
-{% data reusables.actions.minio-gateways-removal %}
-
 ## Prerequisites
 
-Before enabling {% data variables.product.prodname_actions %}, make sure you have completed the following steps:
+Before enabling {% data variables.product.prodname_actions %} on {% data variables.product.product_location_enterprise %}, make sure you have completed the following steps:
+
 
-* To avoid resource contention on the appliance, we recommend that MinIO be hosted separately from {% data variables.location.product_location %}.
+* You need to prepare your MinIO storage bucket. To help you quickly set up a MinIO bucket and navigate MinIO's customization options, see the "[Quickstart for configuring your MinIO storage bucket for {% data variables.product.prodname_actions %}](/admin/github-actions/enabling-github-actions-for-github-enterprise-server/quickstart-for-configuring-your-minio-storage-bucket-for-github-actions)."
+* To avoid resource contention on the appliance, we recommend that MinIO be hosted separately from {% data variables.product.product_location %}.
```

Comment:
Fixing this variable:

```suggestion
* To avoid resource contention on the appliance, we recommend that MinIO be hosted separately from {% data variables.location.product_location %}.
```